.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+## Description
+
+<!-- A brief description of the PR here. -->
+
+## Adding a new package?
+
+If this pull request adds a new package:
+
+- [ ] The package's schema URL is correct.
+- [ ] The package metadata file, if present, contains: 
+  - [ ] a supported category (one of `Cloud`, `Infrastructure`, `Network`, `Database`, `Monitoring`, or `Utility`).
+  - [ ] a description that explains what the package does.
+  - [ ] a valid logo URL to an SVG whose dimensions conform to the others in this repo.
```

Comment:
Is this about the `logoURL` in the Pulumi schema file? If so, SVG is not supported as the icon for a .NET nuget package. If you do, the publishing will fail with this error:

`error: Response status code does not indicate success: 400 (Unsupported icon image format. Only PNG and JPG images are supported.).`

https://github.com/pulumiverse/pulumi-talos/actions/runs/6596863737/job/17923421848

Reply:
Ah yes, PNG works. Thanks for pointing that out!